app/forms/webauthn_setup_form.rb

```diff
@@ -43,6 +43,22 @@ def platform_authenticator?
     !!@platform_authenticator
   end
 
+  def error_message
```

Comment:
I assume we need this method because of the default `validates` at the top of this file? Since those are still generating messages, I think it'd be better to customize those messages at the time they're created, than to retroactively consider if we want to use or bypass an error message.

We do something like this in `WebauthnVerificationForm`, maybe we can do similar here?

https://github.com/18F/identity-idp/blob/4c96efb7632de6218ab8c1f273bd0b6be2be3107/app/forms/webauthn_verification_form.rb#L10-L15

Reply:
yea, hmm good idea